presto-docs/src/main/sphinx/sql/alter-function.rst

```diff
@@ -0,0 +1,47 @@
+==============
```

Comment:
@kaikalur Can you take a look at the documentation for create / alter / drop function? Thanks!